crates/router/src/configs/settings.rs

```diff
@@ -448,6 +449,7 @@ pub struct Jwekey {
     pub locker_decryption_key1: String,
     pub locker_decryption_key2: String,
     pub vault_encryption_key: String,
+    pub rust_locker_encryption_key: String,
```

Comment:
can we have this as optional, this change could break the deployment if not verified